src/shared/src/types/user-types.ts

```diff
@@ -37,6 +37,7 @@ export interface AuthenticatedUser {
   role: Role;
   defaultTheme?: ThemeName;
   teamAsLeadId?: string;
+  favoriteProjectsId?: number[];
```

Comment:
I think we should probably not send favorite projects along with the user at all tbh, it's too much data to send when it is only needed on one page (this means we dont need to update this type or the transformer at all)

also if we were going to use this we would want to send actual projects themselves, not the ids